lib/manager/pipenv/extract.js

```diff
@@ -43,11 +44,31 @@ function extractFromSection(pipfile, section, registryUrls) {
     return [];
   }
   const specifierRegex = new RegExp(`^${specifierPattern}$`);
-  const deps = Object.entries(pipfile[section])
+  const oneSection = pipfile[section];
+
+  if (section === 'packages') {
+    Object.keys(oneSection).forEach(key => {
+      if (!is.string(oneSection[key]))
+        oneSection[key].version = oneSection[key].version || '*';
+    });
+  }
+
+  const deps = Object.entries(oneSection)
     .map(x => {
       const [depName, requirements] = x;
       let currentValue;
       let pipenvNestedVersion;
+      if ([requirements, requirements.version].find(r => r === '*')) {
+        logger.debug(`Skipping extras dependency. depName: ${depName}`);
+        return {
+          depName,
+          skipReason: 'any-version',
+          currentValue,
+          pipenvNestedVersion,
```

Comment:
This pipenvNestedVersion field doesn't seem useful.

Reply:
Done.